src/darknet.h

```diff
@@ -915,6 +915,18 @@ typedef struct detection{
 	int track_id;
 } detection;
 
+// multi_label_one_anchor_choose_maxiou
```

Comment:
Why is this exposed in darknet.h?  Shouldn't it be in yolo_layer.cpp?

Reply:
it's used to remark struct of mloan below.